test/e2e/framework.go

```diff
@@ -773,6 +780,44 @@ func (data *TestData) restartAntreaControllerPod(timeout time.Duration) (*v1.Pod
 	return newPod, nil
 }
 
+// restartAntreaAgentPod deletes the antrea-agent Pod to force it to be re-scheduled. It then waits
+// for the new Pod to become available, and returns it.
+func (data *TestData) restartAntreaAgentPod(timeout time.Duration) ([]*v1.Pod, error) {
```

Comment:
this seems to delete all Antrea Agent Pods, so I  find the name and the comment quite confusing